backend/views/notifications/assign_task.html

```diff
@@ -6,7 +6,7 @@
 
 <p>
     Date: <%= date.toLocaleString() %></br>
-    <a href="<%= config.indabaClientDomain %>/#/survey/<%= survey.id %>/task/<%= task.id %>"><%- body %>. You are assigned to task.</a>
```

Comment:
That's wrong. The endpoint is like `<indabaClientDomain>/task-review/7/4`, where 7 is the projectId and 4 is the taskId.

Reply:
What is wrong? This was the deleted line. 

So we can't always route to the specific task because a task might be assigned to a user, however that task isn't active yet so the user can't view it, in which case they get an error or something weird. So the best thing to do in this task is to route them to their task page.